pkg/pillar/scripts/device-steps.sh

```diff
@@ -121,7 +120,7 @@ if [ -c $TPM_DEVICE_PATH ] && ! [ -f $CONFIGDIR/disable-tpm ] && [ "$P3_FS_TYPE"
     #It is a device with TPM, and formatted with ext4, setup fscrypt
     echo "$(date -Ins -u) EXT4 partitioned $PERSISTDIR, enabling fscrypt"
     #Initialize fscrypt algorithm, hash length etc.
-    $BINDIR/vaultmgr -c "$CURPART" setupVaults
```

Comment:
This reference to CURPART before it has been set seems to be an existing, unrelated bug. @cshari-zededa AFAICT this would result in vaultmgr hitting
		log.Fatal("Insufficient arguments")
Can you check if this is currently causing falures?

Reply:
I have this code running on a device and vaultmgr has not crashed. Is there a specific case where the crash can be seen?

Reply:
You fixed was was broken before which was calling vaultmgr -c "" setupVaults
Maybe the double-quotes was the savior. Anyhow your code is fine.

Reply:
Yes, earlier due to the bug, it was passing an empty string as CURPART, and because of that vaultmgr did not crash.